changelog/README.rst

```diff
@@ -1,44 +1,149 @@
-=========
-Changelog
-=========
+***************
+Changelog Guide
+***************
 
-.. note::
+.. This directory contains "news fragments" which are files that contain
+   a short description of the changes that will be added to the
+   changelog for the next release.
 
-    This README was adapted from the SunPy changelog readme under the terms of
-    the BSD 2-Clause licence.
+.. The rendered version of this document is in PlasmaPy's online
+   documentation at:
+   https://docs.plasmapy.org/en/latest/development/changelog_guide.html
 
-This directory contains "news fragments" which are short files that contain a
-small **ReST**-formatted text that will be added to the next ``CHANGELOG``.
+A changelog tells users and contributors what notable changes have been
+made between each release. Pull requests to PlasmaPy need changelog
+entries before they can be merged, except when the changes are very
+minor. PlasmaPy uses towncrier_ to convert the changelog entries into
+the full changelog. An example changelog entry would be:
 
-The ``CHANGELOG`` will be read by users, so this description should be aimed at
-PlasmaPy users instead of describing internal changes which are only relevant
-to the developers.
+.. code-block:: rst
 
-Make sure to use full sentences with correct case and punctuation, for example:
+   Added a page in the contributor guide that describes how to add
+   changelog entries.
 
-    Add another Braginskii transport coefficient to `plasmapy.transport`.
+Adding a changelog entry
+========================
 
-Please try to use Sphinx intersphinx using backticks.
+Please follow these steps to add a changelog entry after submitting a
+pull request to PlasmaPy's ``main`` branch.
 
-Each file should be named like ``<PULL REQUEST>.<TYPE>.rst``, where ``<PULL
-REQUEST>`` is a pull request number and ``<TYPE>`` is one of:
+#. In the :file:`changelog` directory, create a new file entitled
+   :file:`⟨number⟩.⟨type⟩.rst`, where ``⟨number⟩`` is replaced with the
+   pull request number and ``⟨type⟩`` is replaced with one of the
+   following changelog types:
```

Comment:
Interesting!  This does seem like a useful thing to include since it does make creating a changelog entry a little bit easier.

- [ ] Discuss that we can use `towncrier create --edit`